frappe/public/js/frappe/ui/capture.js

```diff
@@ -234,7 +234,7 @@ frappe.ui.Capture = class {
 
 	setup_remove_action() {
 		let me = this;
-		let elements = this.$template[0].getElementsByClassName("capture-remove-btn");
+		let elements = Array.from(this.$template[0].getElementsByClassName("capture-remove-btn"));
```

Comment:
`for...of` would've worked without the creation of `Array` object. but I've merged the fix, it's valid 👍🏼